tier4_v2x_msgs/msg/ExtraTrafficSignal.msg

```diff
@@ -0,0 +1,7 @@
+std_msgs/Header header
+autoware_perception_msgs/TrafficLightGroup[] states
+bool has_min_rest_time
+float32 min_rest_time
+float32 min_rest_time_to_red
+bool has_max_rest_time
+float32 max_rest_time
```

Comment:
* What is the difference between min_rest_time and min_rest_time_to_red?
* Why max_rest_time does not have max_rest_time_to_red?

Also, it sounds more natural if we name it remaning_time instead of rest_time.